src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import classNames from 'classnames';
+import useStyles from './styles';
+
+type DropdownOption = {
```

Comment:
@Stefafrontani  we are using both interfaces and types indistinctly, we should hang on a little bit and make our reaserch guys.